projects/med_benchmarking/configs/experiment/linear_probing_eval.yaml

```
@@ -0,0 +1,308 @@
+        job_type: train
+    retinamnist:
+      split: train
+      name: retinamnist
+      transform:
+        job_type: train
+    breastmnist:
+      split: train
+      name: breastmnist
+      transform:
+        job_type: train
+    bloodmnist:
+      split: train
+      name: bloodmnist
+      transform:
+        job_type: train
+    tissuemnist:
+      split: train
+      name: tissuemnist
+      transform:
+        job_type: train
+    organamnist:
+      split: train
+      name: organamnist
+      transform:
+        job_type: train
+    organcmnist:
+      split: train
+      name: organcmnist
+      transform:
+        job_type: train
+    organsmnist:
+      split: train
+      name: organsmnist
+      transform:
+        job_type: train
+    nck_crc:
+      split: train
+      transform:
+        job_type: train
+    pad_ufes_20:
+      split: train
+      transform:
+        job_type: train
+    sicap:
+      split: train
+      transform:
+        job_type: train
+  val:
+    pcam:
+      split: test
+      transform:
+        job_type: eval
+    bach:
+      split: test
+      transform:
+        job_type: eval
+    lc25k_lung:
+      split: test
+      transform:
+        job_type: eval
+    lc25k_colon:
+      root_dir: ${oc.env:LC25000_COLON_ROOT_DIR}
+      split: test
+      organ: colon
+      transform:
+        job_type: eval
+    pathmnist:
+      split: test
+      name: pathmnist
+      transform:
+        job_type: eval
+    dermamnist:
+      split: test
+      name: dermamnist
+      transform:
+        job_type: eval
+    octmnist:
+      split: test
+      name: octmnist
+      transform:
+        job_type: eval
+    pneumoniamnist:
+      split: test
+      name: pneumoniamnist
+      transform:
+        job_type: eval
+    retinamnist:
+      split: test
+      name: retinamnist
+      transform:
+        job_type: eval
+    breastmnist:
+      split: test
+      name: breastmnist
+      transform:
+        job_type: eval
+    bloodmnist:
+      split: test
+      name: bloodmnist
+      transform:
+        job_type: eval
+    tissuemnist:
+      split: test
+      name: tissuemnist
+      transform:
+        job_type: eval
+    organamnist:
+      split: test
+      name: organamnist
+      transform:
+        job_type: eval
+    organcmnist:
+      split: test
+      name: organcmnist
+      transform:
+        job_type: eval
+    organsmnist:
+      split: test
+      name: organsmnist
+      transform:
+        job_type: eval
+    nck_crc:
+      split: validation
+      transform:
+        job_type: eval
+    pad_ufes_20:
+      split: test
+      transform:
+        job_type: eval
+    sicap:
+      split: test
+      transform:
+        job_type: eval
+
+dataloader:
+  test:
+    batch_size: 64
+    num_workers: 4
+    shuffle: False
+
+task:
+  task: multiclass
+  num_classes: 4
+  num_output_features: 512
+  hidden_dims: [256]
+  modality: rgb
+  encoder_checkpoint_path: /path/to/checkpoint
```

You can make this a required parameter like so:

```suggestion
  encoder_checkpoint_path: ???
```